config/casestudies/adeo.yml

```diff
@@ -0,0 +1,272 @@
+
+    Shift to using AsyncAPI also enables the team to implement more use cases using AsyncAPI files.
+technical:
+  languages:
+    - Java
+  frameworks:
+    - Spring
+  protocols:
+    - Kafka
+  brokers: |
+    Kafka with Kafka Connect component. There are 15 production brokers with 47 topics. 
+  testing: |
+    For Kafka e2e tests are done with [Zerocode](https://github.com/authorjapps/zerocode). Load tests are handled with [JMeter](https://jmeter.apache.org/) with [kloadgen](https://github.com/corunet/kloadgen) plugin that supports Kafka and Avro.
+  architecture: |
+    The following [enterprise integration patterns](https://www.enterpriseintegrationpatterns.com/patterns/messaging) are applied:
+    - [Request/Reply](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html)
+      Described with `description` field in AsyncAPI. Reply goes to dedicated reply channel.
+      
+      Example description of response channel:
+      ```
+      description: >
+      This topic is used to REPLY Costing Requests and is targeted by the
+      `REPLY_TOPIC` header.
+      ```
+    - [Return Address](https://www.enterpriseintegrationpatterns.com/patterns/messaging/ReturnAddress.html)
+      Info that needs to be provided by client, so producer knows where to send a response. Information is sent in message header with `REPLY_TOPIC` property. In the AsyncAPI file, information is documented as part of Message Header object.
+      
+      Example of request message header with `REPLY_TOPIC`:
+      ```
+      headers:
+        type: object
+        required:
+          - REPLY_TOPIC
+        properties:
+          REPLY_TOPIC:
+            $ref: "#/components/schemas/ReplyTopic"
+      ```
+    - [Correlation Identifier](https://www.enterpriseintegrationpatterns.com/patterns/messaging/CorrelationIdentifier.html)
+      This pattern enables identification of request given response was sent to. 
+      
+      `REQUEST_ID` property is in the request message header. `CORRELATION_ID` property is in response message header. Both headers are described in the AsyncAPI Message Header object and referred to in the AsyncAPI `correlationID` property.
+      This means that correlation identifier is represented by different property in the message header, depending if it is a request or reply.
+      
+      Example of request message header with `REQUEST_ID`:
+      ```
+      headers:
+        type: object
+        required:
+          - REQUEST_ID
+        properties:
+          REQUEST_ID:
+            $ref: "#/components/schemas/RequestId"
+      ```
+
+      Example of how `correlationId` points to `REQUEST_ID`:
+      ```
+      description: >
+        This correlation ID is used for message tracing and messages
+        correlation.
+        This correlation ID is generated at runtime based on the `REQUEST_ID`
+        and sent to the RESPONSE message.
+      location: $message.header#/REQUEST_ID
+      ````
```

Comment:
```suggestion
      ```
  correlationId:
      description: >
        This correlation ID is used for message tracing and messages
        correlation.
        This correlation ID is generated at runtime based on the `REQUEST_ID`
        and sent to the RESPONSE message.
      location: $message.header#/REQUEST_ID
      ````
```